docs/permissions/getting-started.md

```diff
@@ -20,20 +14,12 @@ The permissions framework depends on a few other Backstage systems, which must b
 
 The permissions framework itself is new to Backstage and still evolving quickly. To ensure your version of Backstage has all the latest permission-related functionality, it’s important to upgrade to the latest version. The [Backstage upgrade helper](https://backstage.github.io/upgrade-helper/) is a great tool to help ensure that you’ve made all the necessary changes during the upgrade!
```

Comment:
> The permissions framework itself is new to Backstage and still evolving quickly.

Is this still relevant?

Reply:
Haha, maybe needs rewording, I don't think its considered `new` anymore or evolving `quickly` but I think its still probably evolving as most things are 😅

Reply:
> but I think its still probably evolving as most things are 

Everything is evolving quickly 🤣 I don't know if that's specific to permissions at this point.

Reply:
Yeah it's been out so long that this does seem unnecessary - i guess it's not only about the framework though, but also about every single individual plugin starting to support permissions at their own pace. Could probably remove only the first sentence but keep the rest I guess 🤔 because the other wording kinda encompasses all Backstage things, not just the core